Example Apps/Example Swift/ButtonManager.swift

```diff
@@ -32,11 +32,11 @@ class ButtonManager: NSObject {
         }
     }
 
-    fileprivate var isSubtleAlertAllowed: Bool {
+    private var isSubtleAlertAllowed: Bool {
         return self.sdlManager.permissionManager.isRPCNameAllowed(SDLRPCFunctionName.subtleAlert)
     }
 
-    fileprivate var isAlertAllowed: Bool {
+    private var isAlertAllowed: Bool {
         return self.sdlManager.permissionManager.isRPCNameAllowed(SDLRPCFunctionName.alert)
     }
```

Comment:
What about the rest of the recommended changes?

Reply:
I'm not sure what you are referring to? I don't see any other comments on the Example Apps.

Reply:
I unresolved one above because it wasn't fully resolved, only the fileprivate / private part of it was.